dashboard/test/ui/README.md

```diff
@@ -67,13 +67,9 @@ Run all UI tests using the local chromedriver against your localhost. Faster tha
 
 `./runner.rb -l`
 
-Run all UI tests for a given browser/os combination - full list of combinations is in browsers.json
-
-`./runner.rb --config ChromeLatestWin7`
-
 Run all UI tests for a given browser
 
-`./runner.rb --browser Chrome`
+`./runner.rb --config Chrome`
```

Comment:
`--config or -c` is the flag that will allow you [specify a single browser](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/ui/runner.rb#L269) -- I'm not exactly sure what the `browser` flag does, but this results in running in all browsers for me.